<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
